src/golang/lib/lib_utils/utils.go

```diff
@@ -213,6 +214,32 @@ func ParseEmailConfig(conf auth.Config) (*shared.EmailConfig, error) {
 	}, nil
 }
 
+const (
```

Comment:
I feel this `lib_utils` library is really badly named. We should remove stuffs out and avoid adding new stuffs if possible. Is there a better existing library (maybe `/notification`) we can put this?

Reply:
hmm doing so seems to induce an import cycle, I think `/notification` is at too high of a layer right now. I've moved it into a separate file in `lib_utils` called `notification.go` for now. Agreed this name sucks.

Reply:
hmmm yeah let's keep it in this lib for now